media/memory-inspector.css

```diff
@@ -129,34 +99,62 @@
 }
 
 .advanced-options-toggle {
-  cursor: pointer;
-  display: flex;
-  flex-flow: row nowrap;
-  align-items: center;
+  margin-left: auto;
+}
+
+.advanced-options-toggle:after { content: ''; position: absolute; width: 30px; height: 30px; transform: translateX(-50%) translateY(-25%); }
+.advanced-options-content {
+  position: absolute;
+  z-index: 1;
+  margin: 20px -165px;
+  padding: 12px;
+  border-radius: 1px;
+  background-color: var(--vscode-dropdown-background);
+  box-shadow: var(--vscode-dropdown-border) 0 2px 4px;
+  text-align: left;
+  display: none;
+}
+.advanced-options-content > vscode-button { background-color: var(--vscode-dropdown-background); color: var(--vscode-foreground);}
+.advanced-options-content > vscode-button:hover { background-color: var(--button-primary-hover-background); color: var(--button-primary-foreground); }
+.advanced-options-toggle:focus .advanced-options-content, .advanced-options-content:active, .advanced-options-content:focus-within { display: flex; flex-direction: column; }
+
+.options-dropdown {
+  width: 70px;
+  min-width: 70px;
+}
+
+.options-label {
+  margin: 10px 0px 2px 0px;
+  font-weight: bold;
 }
 
-.advanced-options-toggle i.codicon {
-  padding-right: 3px;
+.multi-select-label {
+  margin-bottom: 2px;
+  font-weight: bold;
 }
 
 .core-options {
-  display: grid;
-  column-gap: 6px;
-  grid-template-columns: 4fr 4fr 2fr 1fr;
+  display: flex;
+  flex-direction: row;
+  align-items: end;
+  column-gap: 12px;
   margin: 6px 0;
 }
 
+.core-options #address {
+  width: 100px;
+}
```

Comment:
In these cases, as well, these specifications are basically overridden by the `160px` on the `multi-select` class, since that expands the width of the parent element.